lib/liquider/ast.rb

```diff
@@ -69,16 +71,16 @@ def visit(visitor)
   CaseElseNode = Node.new_type(:case_else, :body)
 
   ForNode = Node.new_type(:for, :binding, :expression, :body) do
-    def reversed?
-      @reversed ||= options[:reversed] || Ast::BooleanNode.new(false)
+    def reversed
+      options[:reversed] || Ast::BooleanNode.new(false)
```

Comment:
Reversed doesn't support being passed as an option (`reversed: true`) so should it really be a node instead of a literal?